### PR TITLE
Duplicating 2 strings for better i18n

### PR DESCRIPTION
**Pull request by ghost**
Addressing the [issue opened on Transifex by MarchRPG](https://www.transifex.com/habitrpg/habitrpg/translate/#ru/$/21450971?issue=yes):

> This string is used in two places, once as 'pets found: X' and once as 'X pets found'. Can that be standardized to a single usage?
> 
> Now, because Dutch capitalisation rules differ from English, I'm either stuck with the ugly 'X Pets found' and the correct 'Pets found: X' or the nice 'X pets found' and the incorrect 'pets found: X'.

I believe duplicating a sting is better then trying to change the way the lines have been used. So this PR suggests duplicating strings:
- "Pets Found"
- "Mounts Tamed"

used in inventory and in stats with different order of numbers - words

Screenshot showing new (duplicated) strings with \* (asterisk is not present in the PR)
Screenshot

### CODE REVIEW

File: common/locales/en/pets.json [hunk 1]
```diff
@@ -1,10 +1,12 @@
 {
     "pets": "Pets",
     "petsFound": "Pets Found",
+    "petsFoundEnding": "Pets Found",
     "rarePets": "Rare Pets",
     "questPets": "Quest Pets",
     "mounts": "Mounts",
     "mountsTamed": "Mounts Tamed",
+    "mountsTamedEnding": "Mounts Tamed",
```
Comment (crookedneighbor) [inline common/locales/en/pets.json]: If this is important, I feel like we should just construct the entire string. This should also help for cases where the ordering of numbers is different than in English. 

String:

``` json
"stableMountMasterProgress": "Mount Master Progress: <%= number => tamed"
```

Jade file:

``` jade
h4 {{:: env.t('stableMountMasterProgress', { number: mountMasterProgress }) }}
```

**Comment by ghost** [comment]
@crookedneighbor I agree, but I haven't been sure the string keys are not connected with the js functions with the same names

**Comment by ghost** [comment]
@crookedneighbor thanks for your help with binding - you know I wouldn't have done it myself =) 
- would you like the strings to keep the previous formatting: `Beast Master Progress: 0/90 Pets Found` and `Mount Master Progress: 0/90 Mounts Tamed`?
- ~~As I'm testing it now it seems that the binding results in the counter not responding to the changes made on the page. E.g. I have 2/90 pets and feed one of them into a mount - the counter will still show I have 2/90 pets unless I refresh the page - which may be misleading to newbies.~~

**Comment by ghost** [comment]
@crookedneighbor oooops! I've just remembered it's the way Master achievements work! Sorry about it.

**Comment by crookedneighbor** [comment]
Thanks! Notated toward your next tier.

**Pull request merged**